src/components/policy/policy_regular/include/policy/policy_table/types.h

```diff
@@ -247,6 +248,7 @@ struct RpcParameters : CompositeType {
  public:
   HmiLevels hmi_levels;
   Optional<Parameters> parameters;
+  Optional<Boolean> encryption_required;
```

Comment:
The proposal only mentions putting the encryption_required flags in the ApplicationParameters and the functional group parameters. This change has it within the RPC parameters? I do not think this is correct. Having encryption_required on the rpc level was not documented in the proposal.

Reply:
> Alternative solution : add a Boolean flag encryption_required to each RPC to indicate whether the RPC needs protection or not. For example,

This was an alternate solution and should not be implemented in this feature.

Reply:
Right, removed in 45f0888b531adde8ac91715a3c526c75c7c23ba3